src/platform/forms-system/src/js/patterns/array-builder/arrayBuilderText.js

```diff
@@ -57,32 +59,32 @@ export const DEFAULT_ARRAY_BUILDER_TEXT = {
     }’s information before continuing.`,
   editSaveButtonText: props => `Save and Continue`,
   getItemName: itemData => itemData?.name,
-  removeDescription: props => {
+  deleteDescription: props => {
     const itemName = props.getItemName(props.itemData);
     return itemName
-      ? `This will remove ${itemName} and all the information from your list of ${
+      ? `This will delete ${itemName} and all the information from your list of ${
           props.nounPlural
         }.`
-      : `This will remove this ${
+      : `This will delete this ${
           props.nounSingular
         } and all the information from your list of ${props.nounPlural}.`;
   },
-  removeNeedAtLeastOneDescription: props =>
-    `If you remove this ${
+  deleteNeedAtLeastOneDescription: props =>
+    `If you delete this ${
       props.nounSingular
     }, we’ll take you to a screen where you can add another ${
       props.nounSingular
     }. You’ll need to list at least one ${
       props.nounSingular
     } for us to process this form.`,
-  removeNo: props => `No, keep this ${props.nounSingular}`,
-  removeTitle: props => {
+  deleteNo: props => `No, keep this ${props.nounSingular}`,
+  deleteTitle: props => {
     const itemName = props.getItemName(props.itemData);
     return itemName
-      ? `Are you sure you want to remove ${itemName}?`
-      : `Are you sure you want to remove this ${props.nounSingular}?`;
+      ? `Are you sure you want to delete ${itemName}?`
+      : `Are you sure you want to delete this ${props.nounSingular}?`;
```

Comment:
These two should just be

`Delete ${itemName}'s information?`
`Delete this ${props.nounSingular}?`

At some point content deleted "are you sure you want to" from figma and I didn't get that communicated to you.